openwisp_utils/admin_theme/static/admin/js/menu.js

```diff
@@ -194,7 +194,6 @@ function setMenuToggleText() {
 
 function setMenu() {
   setMenuState();
-  setMenuToggleText();
   setTimeout(function () {
```

Comment:
Because add this moment menu changes its state according to user choice and it looks weird due to transition duration. So adding transition after some time.